corehq/util/couch.py

```diff
@@ -18,18 +19,35 @@ def get_document_or_404(cls, domain, doc_id, additional_doc_types=None):
     allowed_doc_types = (additional_doc_types or []) + [cls.__name__]
     try:
         unwrapped = cls.get_db().get(doc_id)
-    except ResourceNotFound:
-        raise Http404()
+    except ResourceNotFound as e:
```

Comment:
Did you mean to pass this to something?  Looks like `traceback.format_exc()` uses this value anyways, is that right?